skll/experiments.py

```diff
@@ -675,23 +679,29 @@ def _classify_featureset(args):
         # models when we're not.
         task_results = None
         if task == 'cross_validate':
-            logger.info("Cross-validating")
+            logger.info('Cross-validating')
             (task_results,
              grid_scores,
              grid_search_cv_results_dicts,
-             skll_fold_ids) = learner.cross_validate(train_examples,
-                                                     shuffle=shuffle,
-                                                     stratified=stratified_folds,
-                                                     prediction_prefix=prediction_prefix,
-                                                     grid_search=grid_search,
-                                                     grid_search_folds=grid_search_folds,
-                                                     cv_folds=cv_folds,
-                                                     grid_objective=grid_objective,
-                                                     output_metrics=output_metrics,
-                                                     param_grid=param_grid,
-                                                     grid_jobs=grid_search_jobs,
-                                                     save_cv_folds=save_cv_folds,
-                                                     use_custom_folds_for_grid_search=use_folds_file_for_grid_search)
+             skll_fold_ids,
+             models) = learner.cross_validate(train_examples,
+                                              shuffle=shuffle,
+                                              stratified=stratified_folds,
+                                              prediction_prefix=prediction_prefix,
+                                              grid_search=grid_search,
+                                              grid_search_folds=grid_search_folds,
+                                              cv_folds=cv_folds,
+                                              grid_objective=grid_objective,
+                                              output_metrics=output_metrics,
+                                              param_grid=param_grid,
+                                              grid_jobs=grid_search_jobs,
+                                              save_cv_folds=save_cv_folds,
+                                              save_cv_models=save_cv_models,
+                                              use_custom_folds_for_grid_search=use_folds_file_for_grid_search)
+            if models:
+                for index, m in enumerate(models):
```

Comment:
do we want the fold models index to start from 0 or 1?

Reply:
I can switch it to 1 unless anybody has a strong preference for starting at 0? I guess the CV results start at fold 1.

Reply:
Yeah, I think matching the index to whatever the results shows makes sense.

Reply:
I've made the change.